app/views/merchants/show.html.erb

```diff
@@ -79,27 +83,36 @@
         </thead>
 
         <tbody>
-
-        <% exiting_order_items = @merchant.existing_order_items_by_merchant %>
-        <% exiting_order_items.each do |item| %>
-          <tr>
-            <td><%= link_to item.order.id, order_path(item.order.id) %></td>
-            <td><%= link_to item.product.title, product_path(item.product.id) %></td>
-            <td><%= item.quantity %></td>
-            <td><%= item.product.price %></td>
-            <td><%= item.subtotal %></td>
-            <td><%= item.created_at.to_s.to_time.strftime('%B %e, %Y, %T') %></td>
-            <td><%= item.order.status%></td>
-            <% if item.shipped %>
-              <td>Shipped</td>
-            <% elsif item.order.status == "pending" || !item.order.status %>
-              <td>Should be paid first</td>
-            <% else %>
-              <td><%= link_to "Ship", ship_item_path(item.id), method: :post, class: "btn btn-primary" %></td>
+        
+        <% exiting_order_items = @merchant.find_orders_by_status(params["Status"]) %>
```

Comment:
You might want to rename to existing_order_items :)

Reply:
Right. Thanks for spotting that!